services/licence-finder/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: '3.7'
+
+x-licence-finder: &licence-finder
+  build:
+    context: .
+    dockerfile: Dockerfile.govuk-base
+  image: licence-finder
+  volumes:
+    - ${GOVUK_ROOT_DIR:-~/govuk}:/govuk:delegated
+    - home:/home/build
+  working_dir: /govuk/licence-finder
+
+services:
+  licence-finder-lite: &licence-finder-lite
+    <<: *licence-finder
+    privileged: true
+    depends_on:
+      - mongo
+      - elasticsearch6
+    environment:
+      MONGODB_URI: "mongodb://mongo/licence-finder_development"
+      VIRTUAL_HOST: licence-finder.dev.gov.uk
+      ELASTICSEARCH_URI: http://elasticsearch6:9200
+      HOST: 0.0.0.0
```

Comment:
I don't think we need these for the 'lite' stack. Might be worth reading [here](https://github.com/alphagov/govuk-docker#stacks).

Reply:
Again, tests until weren't running I added these in.

Reply:
I've managed to run the tests without them (`ports`, `VIRTUAL_HOST` and `HOST`).